src/content/smart-bulletins/030.md

```diff
@@ -0,0 +1,88 @@
+---
+title: "Smart Bulletin No. 030"
+intro: "GSA SmartPay – OMB Circular No. A-123, Appendix B Agency Reporting to GSA/CCCM; Submission Implementing Guidance"
+number: "No. 030"
+order: 030
+businessline: "Purchase, Travel, Fleet and Integrated"
+published: "Dec 18, 2019"
+lastupdated: "Aug 28, 2020"
+---
+
+UPDATE:
+| Date | Version History Action Log - Summary of Action/Changes |
+| ----------- | ----------- |
+| Dec 18, 2019 | Implementation of Smart Bulletin #30. |
+| August 28, 2023 | Revised to update for administrative change and also replaces Smart Bulletin #21. |
+
+## Effective Date
+
+Effective upon issuance and shall remain in force until modified,
+rescinded, or superseded
+
+## Business Line(s) Affected
+
+Purchase, Travel, Fleet and Integrated
+
+## Introduction
+
+The purpose of this GSA SmartPay Smart Bulletin is to provide agencies/organizations with implementing guidance on submitting statistical and narrative reports as required in Chapter 5.3 of Office of Management and Budget (OMB) Circular A-123, Appendix B, entitled “A Risk Framework for Government Charge Card Programs,” dated August 27, 2019. This Smart Bulletin includes: (1) reporting templates for agency submission of chapter 5.3 reports,(2) information on a due date extension solely for FY2019 reporting, and (3) reporting expectations for FY2020 and beyond.
```

Comment:
The purpose of this GSA SmartPay Smart Bulletin is to provide agencies/organizations with implementing guidance on submitting statistical and narrative reports as required in Chapter 5.3 of Office of Management and Budget (OMB) Circular A-123, Appendix B, entitled “A Risk Framework for Government Charge Card Programs,” dated August 27, 2019. This Smart Bulletin includes (1) reporting templates for agency submission of chapter 5.3 reports,(2) information on a due date extension solely for FY2019 reporting, and (3) reporting expectations for FY2020 and beyond.